examples/pylint/W0231_super_init_not_called.py

```diff
@@ -5,4 +5,4 @@ def __init__(self):
 
 class Child(Parent):
     def __init__(self):
-        Parent.__init__(self)  # You must have this
+        pass #Parent.__init__(self)  # You must have this
```

Comment:
You can put the comment on the line above - it's clearer than having it on the same line as the `pass`